man/docker-exec.1.md

```diff
@@ -33,6 +34,13 @@ container is unpaused, and then run
 **-i**, **--interactive**=*true*|*false*
    Keep STDIN open even if not attached. The default is *false*.
 
+**--privileged**=*true*|*false*
+   Give extended privileges to the process to run in a running container. The default is *false*.
+
+   By default, the process run by docker exec in a running container
+have the same capabilities of the container. By setting --privileged will give
```

Comment:
@tjdett thanks for the contribution.  I think @thaJeztah's rewrite still leaves some information ambiguous.  For example, "run in a running" can lose the running because the container has to be running to run a process.  The use of "By default" is confusing given how the previous sentence left off.  It is unfortunate this flag is "privileged" rather than "--capable"  

Try this:

--> https://gist.github.com/moxiegirl/b878e08e7c1cece5dc0c

Give a process extended [Linux capabilities](http://man7.org/linux/man-pages/man7/capabilities.7.html)  when running in a container. The default is *false*.

Without this flag, the process run by `docker exec` in a running container have
the same capabilities of the container, which may be limited. Set
`--privileged=true` to give all capabilities to the process.